config.json

```diff
@@ -112,6 +112,18 @@
             "Configuration": "Release-Intrinsics"
           }
         },
+        "debug-netfx": {
+          "description": "Sets optimization level to debug for managed build configuration and builds against fullframework. (/p:Configuration=Debug-netfx)",
+          "settings": {
+            "Configuration": "Debug-netfx"
```

Comment:
I wouldn't use netfx. I would use the actual target group so that this can scale. If we ever want to run them against another target group we can just add a new configuration. For example the configuration would be `Debug-net461` -- then in the msbuild props I would check if Configuration contains -net, then strip down the framework from what is after the `-` and set it as the `DefaultTestTargetFramework`. @eerhardt what do you think?

Reply:
I choose this name because I though if in future we want to upgrade the version of netfx. we will just need to update  ```<DefaultTestTargetFramework Condition="$(Configuration.Contains('-netfx'))">net462</DefaultTestTargetFramework>```

But yeah if there is possibility that we want to run for different versions of netfx at the same time or in ci, then we may have to change some things.

Reply:
Agree, to upgrade the current version it is easier the way yo have it, but if we end up choosing multiple versions then we will have to do what I suggested. Disregard then.